libs/designsystem/src/lib/components/action-sheet/action-sheet.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { fakeAsync, tick } from '@angular/core/testing';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { MockComponent } from 'ng-mocks';
+
+import { PlatformService } from '../..';
+import { OpenState } from '../../models';
+import { WindowRef } from '../../types';
+import { ButtonComponent } from '../button/button.component';
+import { IconComponent } from '../icon';
+import { ModalController } from '../modal';
+
+import { ActionSheetComponent, ActionSheetPopoutComponent } from './';
+
+describe('ActionSheetComponent', () => {
+  let spectator: SpectatorHost<ActionSheetComponent>;
```

Comment:
If you're not actually modifying the markup when creating the component, consider replacing with:
```suggestion
  let spectator: Spectator<ActionSheetComponent>;
```

Reply:
Fixed